input/docs/handbook/views/extending-iviewfor.md

```diff
@@ -0,0 +1,90 @@
+Title: Extending View Support
+---
+
+# Extending IViewFor
+
+There will be times where you need to extend the power of ReactiveUI view types to concrete implementations you don't control. The answer then is in extending the `IViewFor` interface so that ReactiveUI will pick up and register it against the `ViewLocator` instance.  Once you implement `IViewFor<T>`, binding methods are now available as extension methods on your class.
```

Comment:
ViewModel properties even.

Reply:
We can look at bringing in classes like this.  I explicitly wanted to document the concept because when I was moving from Fresh Mvvm to ReactiveUI, I had to dig around the code to realize I needed to extend `IViewFor` and `IReactiveObject` in order to get the frameworks to play nicely.

Reply:
also just mention the available resources on some platforms already for common base classes.

Reply:
Added additions